lighthouse/classes/events/biosero/source_all_negatives.py

```diff
@@ -1,29 +1,49 @@
 import logging
```

Comment:
This class and file not needed, source_no_pickable_samples is what we called this event for biosero.

Reply:
ok, is removed in the code reviewed branch